selectors.yml

```
@@ -107,18 +146,30 @@ selectors:
   421614:
     selector: 3478487238524512106
     name: "ethereum-testnet-sepolia-arbitrum-1"
+  432201:
+    selector: 1458281248224512906 
+    name: "dexalot-subnet-testnet"
   534351:
     selector: 2279865765895943307
     name: "ethereum-testnet-sepolia-scroll-1"
+  686868:
+    selector: 5269261765892944301 
+    name: "bitcoin-merlin-testnet"
```

the project/network is called `bitcoin-merlin`? Or is `merlin` the name of the bitcoin testnet, in which case it should be named ?
```suggestion
    name: "bitcoin-testnet-merlin"
```